src/main/java/jewellery/inventory/service/OrganizationService.java

```diff
@@ -43,6 +47,15 @@ public OrganizationResponseDto create(OrganizationRequestDto organizationRequest
     return organizationMapper.toResponse(organization);
   }
 
+  @LogDeleteEvent(eventType = EventType.ORGANIZATION_DELETE)
+  public void delete(UUID organizationId) {
```

Comment:
Here also, probably it is better to return something and not just void method. If something happens and for some reason we can't delete we can return for example boolean or our custom object. And then when you call this from controller, based on result you can return either: For positive deletion ( NO CONTENT) and for negative ( NOT FOUND or something else)
This is just a tip to consider

Reply:
So far we've made delete requests return nothing if the request is valid and the actions is successful. If there is a problem with the deletion, we are creating the response in the GlobalExceptionHandler as a result of throwing an exception.